D/DSDP/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "DSDP"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://www.mcs.anl.gov/hs/software/DSDP/DSDP5.8.zip", "8915e55456f1a7cc5c970ad157d094a5fb399737cf192dfe79b89c2d94d97a8a"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd DSDP*
+export DSDPROOT=${PWD}
+
+export DSDPLIB="${DSDPROOT}/lib/libdsdp.a"
+export DSDPLIBSO="${DSDPROOT}/lib/libdsdp.${dlext}"
+
+make DSDPCFLAGS="-Wall -fPIC -DPIC" LAPACKBLAS="-L${libdir} -lopenblas -lm" dsdpapi
+make DSDPCFLAGS="-Wall" LAPACKBLAS="-L${libdir} -lopenblas -lm" RM="rm -rf" SH_LD="${CC} ${CFLAGS} -shared" oshared
+
+mv lib/* $libdir
+mv include/dsdp* $includedir
```

Comment:
Not tested:
```suggestion
DSDPROOT="${prefix}"
DSDPLIB="${DSDPROOT}/lib/libdsdp.a"
DSDPLIBSO="${libdir}/libdsdp.${dlext}"

make DSDPROOT="${DSDPROOT}" DSDPLIB="${DSDPLIB}" DSDPLIBSO="${DSDPLIBSO}" DSDPCFLAGS="-Wall -fPIC -DPIC" LAPACKBLAS="-L${libdir} -lopenblas" dsdpapi
make  DSDPROOT="${DSDPROOT}" DSDPLIB="${DSDPLIB}" DSDPLIBSO="${DSDPLIBSO}" DSDPCFLAGS="-Wall" LAPACKBLAS="-L${libdir} -lopenblas" RM="rm -rf" SH_LD="${CC} ${CFLAGS} -shared" oshared
```
The upstream Makefile is so bad.